build/src/server.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const {spawn} = require('child_process');
+const app = express();
+const port = 8000;
+app.use(bodyParser.json());
+app.use(bodyParser.text());
+
+app.get('/address', (req, res) => {
+
+    let acctScrape = spawn("acctScrape", ['--list', '0x2e18e2d31f14c78a9d1f8bb687c3e0dd71ad6391']);
```

Comment:
In this trashy version,

first edit the Dockerfile so its reading from your local parity node... (uncomment the sed line and comment the other one).

 you can `docker-compose up` and then navigate to http://localhost:8000/address. You won't see anything in the browser. That's because the node api throws an error because acctScrape --list doesn't work yet. You can see what's going on here

Reply:
You'll see the file above is outdated. It now takes an address parameter:
http://localhost:8000/address/0x2e18e2d31f14c78a9d1f8bb687c3e0dd71ad6391